language/tools/vm-genesis/src/lib.rs

```diff
@@ -311,19 +311,6 @@ fn create_and_initialize_testnet_minting(
         account_config::treasury_compliance_account_address(),
         &create_dd_script,
     );
-    exec_function(
```

Comment:
@runtian-zhou: Is deleting this ok?

Reply:
Deleting this means that we won't test this logic in language_e2e_tests, which I think is OK. Because it would be deprecated code.